swift-pipeline

```diff
@@ -1,4 +1,4 @@
-#!python3
+#!/cosma/local/Python/3.6.5/bin/python3
```

Comment:
We don't want to let the script know anything about cosma, so it should be `#!python3`

Reply:
That's true, this is there  because the `#!python3` shebang has never worked for me - `#!/usr/bin/env python3` does work - is there sometihng I'm missing here?

Reply:
I am not fully sure; for me, these two seem to work the same on cosma7

```
[dc-chai1@login7a [cosma7] ~]$ /usr/bin/env python3 --version
Python 3.6.8
[dc-chai1@login7a [cosma7] ~]$ python3 --version
Python 3.6.8
```

Reply:
The python3 shebang should work. If it doesn't then you've got some path linking problems. For the library we have to keep things generic so can't include the COSMA path

Reply:
I have got rid of the cosma references already but have `#!/usr/bin/env python3` in the current version as this also works for me. If you'd like to revert to `#!python3` I can change my local version until I figure out why .